src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 50px;
+}
+
+.nav {
+  display: flex;
+  margin-right: 30px;
+}
+
+.nav__link {
+  font-family: inherit;
+  position: relative;
+  align-items: center;
```

Comment:
The 'align-items' property is used with a flex container, but '.nav__link' is not a flex container. Consider removing it, as it does not have any effect on the element